app/models/solr_document.rb

```diff
@@ -155,7 +155,7 @@ def host
   end
 
   def iiif_manifest_url
-    self[:iiif_manifest_url_ssi] || "/concern/works/#{id}/manifest"
+    self[:iiif_manifest_url_ssi] || "https://iiif.library.ucla.edu/ark%3A%2F21198%2Fzz00090p17/manifest"
```

Comment:
We can't actually merge this, but it's great that it fixed the issue!

Reply:
Probably best to just try `self[:iiif_manifest_url_ssi]` without the `|| ...` bit?

Reply:
substituted:
"https://p-w-dl-viewer01.library.ucla.edu/#?manifest=#{CGI.escape(http_sub(iiif_manifest_url.to_s))}"